lib/ContentTypeParser.js

```diff
@@ -6,11 +6,20 @@ function ContentTypeParser () {
 }
 
 ContentTypeParser.prototype.add = function (contentType, fn) {
+  if (typeof contentType !== 'string') throw new Error('The content type should be a string')
+  if (contentType.length === 0) throw new Error('The content type cannot be an empty string')
+  if (typeof fn !== 'function') throw new Error('The content type handler should be a function')
```

Comment:
```js
assert.ok(typeof contentType !== 'string', 'The content type should be a string')
assert.equal(contentType.length, 0, 'The content type cannot be an empty string')
assert.ok(typeof fn !== 'function', 'The content type handler should be a function')
```

Reply:
It's the same thing, I don't think is needed.

Reply:
Yeah, there is no need to add an extra function invocation for no reason.

Reply:
@jsumners this is not "hot" code so an extra function invocation is not a problem. In any case use `assert` does not give any benefit.
If we want to use it, we should update the entire framework to be more consistent.

Reply:
Yeah. Maybe we'll update the entire code 😨